docs/prometheus_metrics/cluster:ovnkube_metrics.yaml

```diff
@@ -0,0 +1,43 @@
+metrics:
+- labels:
+  - label_description: The original name of the metric as collected by Prometheus.
```

Comment:
Personally, I would prefer not having the `label_` prefix, as it can be already inferred from the parent. The same for the `metric_` prefix below. But I can understand others having other preferences, just my $0.02

Reply:
It was easier for me to locate missing label descriptions vs. missing metrics description this way.
I think this can also be useful for the docs team, which will work manually on this list.

Reply:
Yop, I can see benefits in both ways, might just have a bit different preferences.

Reply:
I would agree with at least having the most unique word in the beginning, reading a yaml file when every line starts with 'label' is hard to visually scan specifically for names and not descriptions

Reply:
Hi @xiormeesh , How do you propose to update the format?

Reply:
For example `name` and `description` so that traversing yaml structures is more in line with k8s naming conventions :
`.metrics.name` instead of `metrics.metric_name`
or
`metrics.labels.name` instead of `metrics.labels.label_name`
so it gets more specific left to right without duplication which makes it easier to read, currently each node starts with either `label` or `metric` which makes it more difficult to get a quick overview of all null descriptions.
![image](https://github.com/user-attachments/assets/854b2564-b95f-4f19-a485-27ab0aaa7808)
But this is a recommendation, if you have a reason to have longer names feel free to ignore